setpath.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+PRODUCT_DIR="$1"
+
+if [ $PRODUCT_DIR = "." ]; then
+    PRODUCT_DIR=$PWD
+fi
+
+echo "Adding: $PRODUCT_DIR to paths"
+export SALPYTOOLS_DIR=$PRODUCT_DIR
+export PYTHONPATH=${PRODUCT_DIR}/python:${PYTHONPATH}
+export PATH=${PRODUCT_DIR}/bin:${PATH}
```

Comment:
I'm not a big fan of using this method for exposing your component... I would use `eups`...